trio/_highlevel_ssl_helpers.py

```diff
@@ -53,6 +53,10 @@ async def open_ssl_over_tcp_stream(
     )
     if ssl_context is None:
         ssl_context = ssl.create_default_context()
+
+        if hasattr(ssl, "OP_IGNORE_UNEXPECTED_EOF"):
+            ssl_context.options ^= ssl.OP_IGNORE_UNEXPECTED_EOF
```

Comment:
Seems like `&= ~ssl.OP_IGNORE_UNEXPECTED_EOF` would be more robust in case upstream changes the default back to what it was before?

Reply:
It looks like you changed the testsuite but not the instance I flagged?

Reply:
AAAH oops I didn't realise I had my find window set to the test file only!